### PR TITLE
MediaGrid Adapter: Put demandSource in bid.meta

### PR DESCRIPTION
**Pull request by patmmccann**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [] Other

## Description of change
Puts demandSource in the correct place: see
https://github.com/prebid/prebid.github.io/pull/1911/files

### CODE REVIEW

**Comment by patmmccann** [comment]
Extends #7179

File: modules/gridBidAdapter.js [hunk 1]
```diff
@@ -357,7 +357,8 @@ function _addBidResponse(serverBid, bidRequest, bidResponses) {
         netRevenue: true,
         ttl: TIME_TO_LIVE,
         meta: {
-          advertiserDomains: serverBid.adomain ? serverBid.adomain : []
+          advertiserDomains: serverBid.adomain ? serverBid.adomain : [],
+          networkName: serverBid.ext && serverBid.ext.bidder && serverBid.ext.bidder.grid && serverBid.ext.bidder.grid.demandSource ? serverBid.ext.bidder.grid.demandSource : ''
```
Comment (ncolletti) [inline modules/gridBidAdapter.js:361]: Any reason this isn't also called demandSource? Also, should this default to 'grid' ?

May make sense to not send the property if 'demandSource' doesn't exist like this:
`...(serverBid.ext && serverBid.ext.bidder && serverBid.ext.bidder.grid && serverBid.ext.bidder.grid.demandSource && {demandSource: serverBid.ext.bidder.grid.demandSource),`

**Comment by ncolletti** [comment]
Could you add a test to cover this and update existing tests to include the new property

File: modules/gridBidAdapter.js (hunk 1 above)
Comment (TheMediaGrid) [reply modules/gridBidAdapter.js:361]: tagging @patmmccann for visibility
I'm assuming networkName vs demandSource is a CafeMedia requirement? we'd rather keep it similar to the actual bid response param though (i.e. demandSource)

as for the default value, right, it should be set to 'grid'

Comment (patmmccann) [reply modules/gridBidAdapter.js]: Network name is documented here https://github.com/prebid/prebid.github.io/pull/1911

Demandsource is not

Comment (TheMediaGrid) [reply modules/gridBidAdapter.js]: @patmmccann I suggest using demandSource instead as it's actually breaking down the network into separate demand sources so we should make sure we don't block the publishers and bidders from using both, what do you think?

Comment (patmmccann) [reply modules/gridBidAdapter.js]: this works for me, i'll do a documentation change as well in that case

**Comment by patmmccann** [comment]
docs pull: https://github.com/prebid/Prebid.js/pull/7184

**Review by ncolletti — APPROVED** [review]

**Pull request merged**